### DISCUSSION TITLE
Almost a year since the last binary release. Is it time for a new one?

### DISCUSSION THREAD

**Discussion by Knezzen** [SpectrumIM/spectrum2#463] [General]
Hi!

Compiling from source is tedious on a production server.
Is it in scope for the near future to compile the current state into 2.1.4 and distribute the files to all the regular repositories?

Thanks for keeping up the good work with Spectrum2!

**Comment by vitalyster** [comment]
You can always try the current state using our Docker container. We don't have ready to use infrastructure to build packages for "regular repositories".
And it is too early to make a release because current state has sensible changes - new accounts.xml behavior and new paths for backend binaries, it may need more work to test these changes and handle possible issues.

**Comment by vitalyster** [comment]
Spectrum 2.2.0 is out!
